airbyte-ci/connectors/metadata_service/lib/poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.4.2 and should not be changed by hand.
```

Comment:
@bnchrch I'm worried that we may not be using the same poetry version. Could you please checkout this branch and check if a `poetry install` works correctly?

Reply:
`poetry install` worked 
https://app.warp.dev/block/MXAlAwrwhk3rO83WmFZ2Zg

`poetry run metadata_service upload --help` worked
https://app.warp.dev/block/pqWT4J1W1yi55eMaAWYKqT

But! my poetry is behind `Poetry (version 1.3.2)`

So I updated to v1.4.2 and the above commands still work.

What was going wrong on your end?

Reply:
Nothing was wrong :D It's just that these changes on the lock file looked suspicious.